stories/atoms/Badges.stories.tsx

```diff
@@ -24,7 +25,19 @@ export const _StatusBadgeWithLeadingIcon: Story = {
     args: {
         children: 'Checked',
         variant: 'bold',
-        leadingIcon: <FaCheck />,
+        leadingIcon: <Check />,
     },
     render: (args) => <StatusBadge {...args} />,
 };
+
+type CounterBadgeStory = StoryObj<typeof CounterBadge>;
+
+export const _CounterBadge: CounterBadgeStory = {
```

Comment:
I tried to see how it looks like with simple content (just 1, no arrow). It doesn't look like expected.
<img width="72" alt="Screenshot 2024-12-03 at 15 27 56" src="https://github.com/user-attachments/assets/0a51527b-fef7-4d87-8fd6-eccc364ea380">


Maybe it is only in storybook, some of the things don't re-render if the props are changed? Not sure. Anyway, might be nice to have a few stories: with arrow; without arrow; with custom trailing icon; custom leading icon?

Reply:
looked closer and it comes from the min-width defined on the StatusBadge. Also there is a fixed hight there, which I'm not sure is the best thing. E.g. if the font size would change, it would look strange.

Note for the future: maybe good to create some CSS vars for heights, that can be conditioned on the font size. I know you prefer not to have height fixed, but I'm not sure we can completely get away with it. But if we have some variables for it, that can help to align things. For instance something like `--form-field-height` (which wasn't used in the previous rotation and gives us quite some headache when trying to put 2 elements next to each other.